index.html

```diff
@@ -234,46 +246,25 @@
-                                <td class="form-td"><input type="text" placeholder="Last Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td colspan="2" class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="first-name" placeholder="First Name">
+                        <input type="text" name="last-name" placeholder="Last Name">
+                        <input id="email" name="e-mail" type="email" placeholder="E mail address">
+                        <td colspan="2" name="message" class="form-td"><textarea>Typing something</textarea>
+                        <button class="see-project-btn">Get in touch</button>
                     </form>
                 </div>
                 <div class="touch-class3">
-                    <form>
-                        <table class="form-table">
-                            <tr class="form-tr couple-tr">
-                                <td class="form-td"><input type="text" placeholder="Full Name"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><input id="email" type="email" placeholder="E mail address"></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td class="form-td"><textarea>Typing something</textarea></td>
-                            </tr>
-                            <tr class="form-tr">
-                                <td align="center"><button class="see-project-btn">Get in touch</button></td>
-                                <td></td>
-                            </tr>
-                        </table>
+                    <form class="mob_form" action="https://formspree.io/f/moqzdnpo" method="post">
+                        <input type="text" name="full-name" maxlength="30" required 
+                         placeholder="Full Name">
+                        <input id="email" name="e-mail" required type="email" placeholder="Email address">
+                        <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+                        <button class="see-project-btn mob_btn">Get in touch</button>
```

Comment:
- [ ] 😃 Kindly make sure, that the `on-focus` property of the `input` elements is set with the design guideline, this will enable you have a pixel-perfect design,

- Your design
<img width="178" alt="wrong2" src="https://user-images.githubusercontent.com/41163571/220490926-b1943a8b-f8c9-4841-b9de-d364df938a34.png">

- Figma design
<img width="173" alt="right" src="https://user-images.githubusercontent.com/41163571/220491040-f3823d60-7f02-4a35-b343-45822f2eb373.png">



Kindly note that this doesn't apply to just the `textarea` element, but rather all the other `input` elements.

Reply:
![image](https://user-images.githubusercontent.com/54537654/220785609-09ccaf9e-ea49-4430-99ce-2656b5d9d631.png)
Now you will see that on hover property is working fine on all form elements.